CHANGELOG.md

```diff
@@ -5,7 +5,15 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 
-## [UnReleased] - 2021-MM-DD
+## [1.2.3] - 2021-MM-DD
+
+### Fixed
+
+
+- Fixed DP reduction with collection ([#6324](https://github.com/PyTorchLightning/pytorch-lightning/pull/6324))
+
+
+## [1.2.2] - 2021-03-03
```

Comment:
Updating the CHANGELOG is a responsability of the releaser. Shouldn't do it here

see https://github.com/PyTorchLightning/pytorch-lightning/pull/6325

Reply:
Resolved !